### PR TITLE
Wip: "Usable" Student Data

### PR DESCRIPTION
**Pull request by IndraA3**
- [x] Some Database Migration Update 95ed9bd7b07d7584541717a0975a9ff1f198a436
- [x] Student Seeder

- [x] Show Student
- [x] Create Student
- [x] Create Student Form Request
- [x] Delete Student

### CODE REVIEW

File: app/Http/Controllers/Datatables/StudentDatatables.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+class StudentDatatables extends Controller
+{
+    public function index()
+    {
+        $students = Student::with('school');
+        return DataTables::of($students)
+                        ->editColumn('gender', function ($data) {
+                            return strtolower($data->gender) == Student::GENDER_LAKI ? 'Laki' : 'Perempuan';
+                        })
```
Comment (backend-sempoa) [inline app/Http/Controllers/Datatables/StudentDatatables.php:18]: Lebih lengkap "Laki-laki" deh. wkwkwkw.

File: resources/views/layout/head-css.blade.php [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
     @yield('title', $title ?? 'undefined') | {{ env('APP_NAME') }}
 </title>
-<link href="vendor/fontawesome-free/css/all.min.css" rel="stylesheet" type="text/css">
+<link href="{{ asset('vendor/fontawesome-free/css/all.min.css') }}" rel="stylesheet" type="text/css">
 <link
     href="https://fonts.googleapis.com/css?family=Nunito:200,200i,300,300i,400,400i,600,600i,700,700i,800,800i,900,900i"
     rel="stylesheet">
 
 <link href="{{ asset('css/sb-admin-2.css') }}" rel="stylesheet">
+<link href="{{ asset('vendor/datatables/dataTables.bootstrap4.min.css') }}" rel="stylesheet">
```
Comment (backend-sempoa) [inline resources/views/layout/head-css.blade.php:10]: untuk asset DataTables, sudah ada di komponen datatable

File: resources/views/pages/students/index.blade.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
+@extends('layout.master-page')
+
+@section('title', 'Students')
```
Comment (backend-sempoa) [inline resources/views/pages/students/index.blade.php:3]: ini udah di-handle via Controller aja

File: resources/views/layout/head-js.blade.php [hunk 4]
```diff
@@ -9,5 +9,13 @@
 <!-- Custom scripts for all pages-->
 <script src="{{ asset('js/sb-admin-2.min.js') }}"></script>
 
+<!-- Page level plugins -->
+<script src="{{ asset('vendor/datatables/jquery.dataTables.min.js') }}"></script>
+<script src="{{ asset('vendor/datatables/dataTables.bootstrap4.min.js') }}"></script>
```
Comment (backend-sempoa) [inline resources/views/layout/head-js.blade.php:14]: untuk asset DataTables, sudah ada di komponen datatable.

**Review by backend-sempoa — CHANGES_REQUESTED** [review]

**Comment by IndraA3** [comment]
some Updates #[a7d39c7](https://github.com/minikopi/sekolah/pull/16/commits/a7d39c7efab903b65957473d5be2a4a709e551d5)

**Review by backend-sempoa — APPROVED** [review]
LGTM

**Pull request merged**